graphql-java-support/src/test/java/com/apollographql/federation/graphqljava/FederationTest.java

```diff
@@ -23,29 +26,42 @@
 
 class FederationTest {
     private final String emptySDL = TestUtils.readResource("schemas/empty.graphql");
+    private final String emptySDLOutput = TestUtils.readResource("schemas/empty-output.graphql");
```

Comment:
Refactoring this to a separate file is a nice change as this is more consistent with the rest of the tests here.

One thing to point out for other reviewers here is that the expected printed schema is different with this version because the include, skip and deprecated directives are now output.